### PR TITLE
* FIX [conf test] a patch to fix CI fail in windows.

### PR DESCRIPTION
**Pull request by OdyWayne**

### CODE REVIEW

File: cmake/NNGHelpers.cmake [hunk 1]
```diff
@@ -111,7 +111,7 @@ function(nng_test NAME)
                 ${PROJECT_SOURCE_DIR}/src
                 ${PROJECT_SOURCE_DIR}/include)
         add_test(NAME ${NNG_TEST_PREFIX}.${NAME} COMMAND ${NAME} -t -v)
-        set_tests_properties(${NNG_TEST_PREFIX}.${NAME} PROPERTIES TIMEOUT 180)
+        set_tests_properties(${NNG_TEST_PREFIX}.${NAME} PROPERTIES TIMEOUT 60)
```
Comment (wanghaEMQ) [inline cmake/NNGHelpers.cmake:114]: Not recommended... This will increase the cost of rebasing upstream

**Review by wanghaEMQ — COMMENTED** [review]
Cool

**Review by wanghaEMQ — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]
DK what happened?

**Comment by wanghaEMQ** [comment]
> DK what happened?

On windows. CI spent too much time on reading the configuration file. And always failed at the end.

**Comment by OdyWayne** [comment]
conf parse has been using a posix api which windows do not support.

**Comment by RanMaoyi** [comment]
> conf parse has been using a posix api which windows do not support.

which api?

**Comment by OdyWayne** [comment]
> > conf parse has been using a posix api which windows do not support.
> 
> which api?

I was wrong, the real problem is that the format of the file path is different in windows.

**Pull request merged**